corehq/apps/locations/resources/v0_5.py

```diff
@@ -49,7 +49,7 @@ class Meta:
         }
 
 
-class LocationResource(SQLResourceURIMixin, ModelResource, HqBaseResource):
```

Comment:
You can probably just delete `SQLResourceURIMixin` and move that method into the other class, since it's only used in one place.